xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -354,6 +354,68 @@ class XAPIAN_VISIBILITY_DEFAULT SVMRanker: public Ranker {
 
 };
 
+class XAPIAN_VISIBILITY_DEFAULT ListMLERanker: public Ranker {
+    /// Ranker parameters
+    std::vector<double> parameters;
+    /// Learning rate (Default is 0.001)
+    double learning_rate;
+    /// Number of iterations (Default is 10)
+    int iterations;
+
+    /** Train the model.
+     *
+     * @exception InvalidArgumentError will be thrown if training_data is
+     *            empty.
+     */
+    void train(const std::vector<Xapian::FeatureVector> & training_data);
+
+    /** Save ListMLE model as db metadata.
+     *
+     *  ListMLE model file gets stored with each parameter value in a new line.
```

Comment:
It's no longer a model file, so this comment needs updating. Just "The model gets stored with…" would be fine.